docs/framework/architecture/ui-library.md

```diff
@@ -357,6 +357,14 @@ dropdownView.buttonView.set( {
 } );
 ```
 
+You can use one of the {@link framework/architecture/ui-components#icons icons available in the editor}. You can also add a custom icon to the dropdown by providing the entire XML string of the icon, like in this example:
```

Comment:
I think we should also mention the possibility of using XML string for `ButtonView` icon here https://ckeditor.com/docs/ckeditor5/latest/framework/architecture/ui-components.html#icon

Reply:
In commit.